apps/modernization-ui/src/generated/schema.graphqls

```diff
@@ -137,6 +137,7 @@ input InvestigationFilter {
   notificationStatuses: [NotificationStatus]
   processingStatuses: [ProcessingStatus]
   reportingFacilityId: ID
+  reportingProviderId: ID
```

Comment:
can you make these two strings?  GraphQL will use these values to cache the individual investigation object which may lead to weird results if investigations have the same reporting facility / provider